src/api/Posts.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable camelcase */
+import { WP_REST_API_Post } from 'wp-types';
+type ApiPost = WP_REST_API_Post;
+/* eslint-enable camelcase */
+
+import { Post, PostContent, PostDate, PostTitle } from '@/model/Post';
+import { ApiClient } from '@/api/ApiClient';
+
+interface CreateBody {
+	guid: string;
+}
+
+interface UpdateBody {
+	date?: PostDate;
+	title?: PostTitle;
+	content?: PostContent;
+}
+
+interface PostMeta {
+	guid: string;
+	raw_title: string;
+	raw_date: string;
+	raw_content: string;
+}
+
+export class PostsApi {
+	// eslint-disable-next-line no-useless-constructor
+	constructor( private readonly client: ApiClient ) {}
+
+	async create( body: CreateBody ): Promise< Post > {
+		const response = ( await this.client.post( '/liberated_posts', {
+			meta: {
```

Comment:
Perhaps this can also accept other fields as we do in `update()`

Reply:
For the moment the only field that the ui passes when creating the post is the guid, but indeed we can add other fields here as the needs evolve.